### DISCUSSION TITLE
Welcome to pdbuzzboard Discussions!

### DISCUSSION THREAD

**Discussion by pdbuzzboard** [pdbuzzboard/pdbuzzboard#3] [Announcements]
This is a discussion board for all things buzzboard and related glove builds.

Feel free to post questions, suggestions, respond as appropriate and celebrate successes!

I will add other discussions on specific topics as needed.  Please feel free to add your own!

**Comment by ke3c** [comment]
Where did you find the specifications for the Stanford design?

**Reply by Richardengle** [reply]
search for STANFORD and PETER TASS and VIBROTACTILE GLOVE.  eventually you will find a link to a paper of his.  click the link and you will either get a way to read the full article or, if not - copy and paste the title to google.  one of the search results will have the full PDF paper to read.

**Comment by werminghausen** [comment]
I have 2 questions  about  Pattern B

Pattern A seems to be straight forward
Pattern B with the 23.5% 'jitter'.... the timely sequence is not so clear, mainly because the graphical representation of the pattern in the research paper might be not very accurate. 
1) jitter: It is hard to determine accurately which positions are shifted causing a shorter or larger pause between fingers (shorter or larger than the standard 0.06666s pause). Please comment on this pattern B attached. 
<img width="376" alt="Screenshot 2023-05-07 at 6 18 14 AM" src="https://user-images.githubusercontent.com/132231504/236679728-83819e83-9071-4bbd-8a4e-95bea5e8bc7b.png">
2) It is stated in the research paper that pattern B was applied 'mirrored' between the 2 hands.... I am guessing that this means the start of the pattern should be synchronized (start of patterns left and right hand at the same time). 

Please comment on the 2 items. 

Best, Martin

**Reply by mkkashyap30** [reply]
Hi there! Where were you able to get the Pattern B code from? I've gone through the research articles & am not able to find them. Trying to build these gloves for my dad who has PD. Thanks in advance :)

**Comment by blazecompton** [comment]
I am interested in your partner's experience when she first started. How many buzzboard sessions did she do before she noticed improvements in her PD symptoms? And, what did she first notice? I have built a buzzboard (yesterday) and have used it 3 times and I need to repair it already because the motor soldering on 2 of the fingers seems to be weak or disconnected. So, in the 3 times I've used it I have noticed no benefits - but that may be because of the motor problems.

**Comment by pdbuzzboard** [comment]
It took several days before I noticed improvements. Her overall mood was probably the first I noticed. Next was a reduction in tremor.Failed motors will definitely impact performance. The first board I built burned up the motors after about 10 days and she backslid over a couple days.On May 10, 2023, at 9:32 PM, blazecompton ***@***.***> wrote:﻿
I am interested in your partner's experience when she first started. How many buzzboard sessions did she do before she noticed improvements in her PD symptoms? And, what did she first notice? I have built a buzzboard (yesterday) and have used it 3 times and I need to repair it already because the motor soldering on 2 of the fingers seems to be weak or disconnected. So, in the 3 times I've used it I have noticed no benefits - but that may be because of the motor problems.

**Comment by blazecompton** [comment]
Thanks for that reply. I'll keep repairing until I notice some results. I'm
hoping that when the commercial version comes out it's affordable and
durable!

**Comment by ghost** [comment]
With the PD buzz board, and with the single controller gloves, you take great care to make sure that both tinger one wires go to motor one, same for  inger two, three and four. With a dual controller design, how do you get both fingers to fire simultaneously?

With dual controller gloves, how do you synchronize the firing of both finger ones, both finger twos both finger threes in both finger fours if there is no physical connection between the two motors?

**Comment by ghost** [comment]
I have created a buzz board, stopped rotary for 1 day and then used the buzzboard for the without meds on the first day, then added back half rotary dosage on the second day after experiencing the usual I’ll effects of abruptly stopping meds.  I am a 62 years old male diagnosed with Parkinson’s and put on Rytary 9 years ago.  Over the years, I have experimented with performing a medication reset by taking a med holiday and it was helpful to improve efficacy of meds.  But the day without meds was no party. Similar experience so far with buzz board. 

Did your wife halt or reduce her meds?  If so, was it before she began first session of buzzboard, as in the Peter Tass study at Stanford?

I will use the buzzboard again today with medication. Keeping my hands on the buzzers for a solid hour without slipping off has been challenging. A family member has volunteered to make some gloves. We will go with the single controller design so that both finger 1s, 2s, 3s and 4s can be fired simultaneously from a single motor per the design used by Stanford 

I am not a physician and am in no way qualified to give medical advice

**Comment by werminghausen** [comment]
Hi, my PD partner has used the gloves for about one week now since I
finished them. She continued taking her meds (no change here) and she did
the glove vCM 2x2  hrs a day. I can report that I don‘t see an immediate
improvement in her tremors and walking, probably the opposite for the first
4/5 days. The last few days she was improving in her mood and tremor
to about where she was before. I am trying to be objective.
I believe there won‘t be a sudden improvement in her case but possibly
enough accumulation of the desynchronizing effect as stated in the paper,
that is the hope.
I am sure there are individual differences as it is true for PD in general.
However  my partner did not see an immediate improvement like the one the
Tass video is showing.
That sudden and strong improvement is possibly too optimistic for most.  It
takes many years for the development of PD and a spontaneous improvement
within a week is probably not realistic for most of the PD patients... that
is what I think.

The glove I built is using Pattern A for the time being. I believe that
pattern A for the left and right hand is not starting simultaneously from
what I read in the paper.
While Pattern B does start simultaneously. Maybe I am mistaken. Please
advise.
Pattern B is programmed as well  and I intend to switch to Pattern B
probably after a month for a test. The left and right glove are then
'mirrored' via bluetooth.

**Reply by Cropsy** [reply]
Anything new with your gloves, any positive symptom relief as of yet?   Still waiting here for more positive consistency.....

**Comment by pdbuzzboard** [comment]
With the micro controller gloves, there is no coordination between 
hands.  Both the two hands together and two hands apart are referenced 
in the docs.  It is unclear which is preferred or gives better results.  
This may be one of the changes they are making for individuals, but the 
documentation doesn't speak to it.

**Comment by pdbuzzboard** [comment]
I am also not a physician and unqualified to give medical advice.  I'm 
just sharing what we have experienced.

She did not drop any medication before starting treatment.  I believe 
(but this is just conjecture) that the Stanford studies dropped meds in 
order to maintain the purity of their study.  Good science but misery 
inducing.  After several days, she was able to see an improvement in 
motor symptoms.  Since that time, we have substantial reduced her blood 
pressure meds (her nOH is pretty debilitating), but have maintained her 
Rytary levels.  Her off times at the end of the Rytary have, 
essentially, disappeared.  (a particularly stressful day will cause her 
to burn through her Rytary before the next dose.  The board helps to 
suppress symptoms, but is not a cure).

When she uses the buzzboard, she normally reclines on the couch and 
rests her fingers on the motors.  This seems to minimize the stress on 
her shoulders, wrists and arms.

Others have also not seen immediate results after a few days and have 
good success by changing to a different software load.

Best of luck and thanks for the update.

**Reply by Richardengle** [reply]
which load seems to work best?

**Comment by pdbuzzboard** [comment]
I have not heard of a worsening of symptoms from using the gloves/board 
before.  Have you tried a different pulse width?

The bluetooth coordination sounds very cool.  Please share code when 
available.

**Comment by Raorathna** [comment]
Hi,  I was able to put together the glove version (with the finger cots).  Thanks for the video it was most helpful.

Operation wise, I noticed the buzzing occurring in random pattern.  However, the buzzing turns off after about 15 minutes.  I have to unplug the power and plug it back on for it to operate it again.  

I am powering it via a battery pack which gives out 5v out put.  The motors are 3v devices.  Can any of you specify which power source is to be used?  Thanks.

**Comment by pdbuzzboard** [comment]
I have had success with 5v 2amp battery packs that are sold to charge phones.Are both hands stopping after 5 minutes?On Jun 4, 2023, at 10:02 AM, Raorathna ***@***.***> wrote:﻿
Hi,  I was able to put together the glove version (with the finger cots).  Thanks for the video it was most helpful.
Operation wise, I noticed the buzzing occurring in random pattern.  However, the buzzing turns off after about 15 minutes.  I have to unplug the power and plug it back on for it to operate it again.
I am powering it via a battery pack which gives out 5v out put.  The motors are 3v devices.  Can any of you specify which power source is to be used?  Thanks.

**Reply by Raorathna** [reply]
Thanks for your reply.   I will let you know about the device stopping if it does so consistently.  Regards.

**Comment by MStalGA** [comment]
The link in the excel file for the 15,000 rpm motor was actually to a 12,000 rpm motor...unless I am looking at it incorrectly.  Could you please provide an updated link to a correct 15,000 rpm motor for this application? I would greatly appreciate it! I am trying to build a buzz board for my mom. 

Here is the link that was in the excel spreadsheet of the buzzbaord video:

https://www.digikey.com/en/products/detail/vybronics-inc/VW0625AB001G/9974285?utm_medium=email&utm_source=oce&utm_campaign=4251_OCE23RT&utm_content=productdetail_US&utm_cid=3495186&so=79659967&mkt_tok=MDI4LVNYSy01MDcAAAGJeLANVy54r4s09P0NdLuKXp0oQGAFKMkSKoxt_WUR0HqpIuFA4PdEmN4esEYX0BRe8__EBr-lKqzRIeW9mk7AxVl_82RnD_y7rBz7el3k

**Comment by pdbuzzboard** [comment]
Vybronics has down rated the motors to 12k. Initial results from a number of users is that they are giving good results.On Dec 8, 2023, at 5:26 PM, MStalGA ***@***.***> wrote:﻿
The link in the excel file for the 15,000 rpm motor was actually to a 12,000 rpm motor...unless I am looking at it incorrectly.  Could you please provide an updated link to a correct 15,000 rpm motor for this application? I would greatly appreciate it! I am trying to build a buzz board for my mom.
Here is the link that was in the excel spreadsheet of the buzzbaord video:
https://www.digikey.com/en/products/detail/vybronics-inc/VW0625AB001G/9974285?utm_medium=email&utm_source=oce&utm_campaign=4251_OCE23RT&utm_content=productdetail_US&utm_cid=3495186&so=79659967&mkt_tok=MDI4LVNYSy01MDcAAAGJeLANVy54r4s09P0NdLuKXp0oQGAFKMkSKoxt_WUR0HqpIuFA4PdEmN4esEYX0BRe8__EBr-lKqzRIeW9mk7AxVl_82RnD_y7rBz7el3k

**Comment by Cropsy** [comment]
Have you found any 15k motors as I am looking as well.  Suppose to have called Digi-Key recently to check, but Parkinsons apathy got the better of me.
   
  
    
   
  
   I will call and will provide the link if I find it first.  Thanks for including me on these emails.
   
  
    
   
  
   Best,
   
  
   Eric
   
   
   
    On 12/11/2023 12:25 PM EST pdbuzzboard ***@***.***> wrote:
    
   
     
    
   
     
    
     Vybronics has down rated the motors to 12k. Initial results from a number of users is that they are giving good results.On Dec 8, 2023, at 5:26 PM, MStalGA ***@***.***> wrote: 
   The link in the excel file for the 15,000 rpm motor was actually to a 12,000 rpm motor...unless I am looking at it incorrectly. Could you please provide an updated link to a correct 15,000 rpm motor for this application? I would greatly appreciate it! I am trying to build a buzz board for my mom. 
   Here is the link that was in the excel spreadsheet of the buzzbaord video: 
   https://www.digikey.com/en/products/detail/vybronics-inc/VW0625AB001G/9974285?utm_medium=email&utm_source=oce&utm_campaign=4251_OCE23RT&utm_content=productdetail_US&utm_cid=3495186&so=79659967&mkt_tok=MDI4LVNYSy01MDcAAAGJeLANVy54r4s09P0NdLuKXp0oQGAFKMkSKoxt_WUR0HqpIuFA4PdEmN4esEYX0BRe8__EBr-lKqzRIeW9mk7AxVl_82RnD_y7rBz7el3k

**Reply by MStalGA** [reply]
Unfortunately I don't know enough about electronics to properly select another motor myself.  Please let me know if you figure out an appropriate 15,000 rpm one.  Otherwise, I may just use the 12,000 rpm one for now.

**Comment by Cropsy** [comment]
I sent an email to Digi-Key and will follow up and keep you posted.  Hoping to have an understandable response this week.
   
  
    
   
  
   Eric
   
   
   
    On 12/16/2023 1:22 PM EST MStalGA ***@***.***> wrote:
    
   
     
    
   
     
    
     
   Unfortunately I don't know enough about electronics to properly select another motor myself. Please let me know if you figure out an appropriate 15,000 rpm one. Otherwise, I may just use the 12,000 rpm one for now.

**Comment by gurdy4me** [comment]
New to this discussion, and to Github, so apologies if I'm not following the protocol...

I think I've found a DigiKey link which appears to offer Vybronics 3v, 15k rpm motors:

https://www.digikey.com/en/products/detail/vybronics-inc/VCDM1027B003L/12323582

However, they are described as "brushed," while the ones in the original link are "brushless." Not sure if it makes a difference?  Also, the original link showing the now down-rated 12k rpm motors lists them as out of stock...

~ Matt


> Have you found any 15k motors as I am looking as well.  Suppose to have called Digi-Key recently to check, but Parkinsons apathy got the better of me.   I will call and will provide the link if I find it first.  Thanks for including me on these emails.   Best, Eric On 12/11/2023 12:25 PM EST pdbuzzboard ***@***.***> wrote:       Vybronics has down rated the motors to 12k. Initial results from a number of users is that they are giving good results.On Dec 8, 2023, at 5:26 PM, MStalGA ***@***.***> wrote: The link in the excel file for the 15,000 rpm motor was actually to a 12,000 rpm motor...unless I am looking at it incorrectly. Could you please provide an updated link to a correct 15,000 rpm motor for this application? I would greatly appreciate it! I am trying to build a buzz board for my mom. Here is the link that was in the excel spreadsheet of the buzzbaord video: https://www.digikey.com/en/products/detail/vybronics-inc/VW0625AB001G/9974285?utm_medium=email&utm_source=oce&utm_campaign=4251_OCE23RT&utm_content=productdetail_US&utm_cid=3495186&so=79659967&mkt_tok=MDI4LVNYSy01MDcAAAGJeLANVy54r4s09P0NdLuKXp0oQGAFKMkSKoxt_WUR0HqpIuFA4PdEmN4esEYX0BRe8__EBr-lKqzRIeW9mk7AxVl_82RnD_y7rBz7el3k —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

**Reply by MStalGA** [reply]
Do those links work for you?  They seem to be bad links on my end?

**Reply by gurdy4me** [reply]
Sorry MStalGA, the link I quoted does appear to work, but the new link I posted was bad!  I fixed it.... but here it is again:

https://www.digikey.com/en/products/detail/vybronics-inc/VCDM1027B003L/12323582
